roles/sahara/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+sahara:
+  enabled: True
```

Comment:
@dlundquist - The only reason I created new vagrant environment was https issue and networking with Sahara. For some reason Sahara is not working properly with self signed certificates and I had to change all endpoints to http, which was done in separate environment. 

Moreover, I made changes in networking section as well of defaults.yml so I wasn't sure you need that or not hence new environment. Once I solve https issue and you are happy with networking change this can be removed.